amethyst_ui/src/event.rs

```diff
@@ -48,13 +62,13 @@ impl UiEvent {
 /// Will only work if the entity has a UiTransform component attached to it.
 /// Without this, the ui element will not generate events.
 #[derive(Default, Serialize, Deserialize, Clone)]
-pub struct MouseReactive;
+pub struct Interactable;
```

Comment:
So, `Interactable` now means that the entity can react to both mouse and keyboard events? Or at least, it will in the future? Maybe it make sense to keep mouse in the name somehow, since it might be preferable if we can split keyboard and mouse events

Reply:
Interactable for both mouse and keyboard. If you have `Selected` a checkbox and press the space key, it will count as either a `Click` or as a `PressAction`, depending on the path we decide to take with eventing.